### PR TITLE
Add deploy to github on tag

### PR DESCRIPTION
**Pull request by betatim**
With this setup travis should auto deploy when we create a tag, and use the name of the tag as version.

### CODE REVIEW

**Comment by rgbkrk** [comment]
:eagle:

File: .travis.yml [hunk 1]
```diff
@@ -5,18 +5,36 @@ notifications:
...
 matrix:
   include:
-    - os: linux
-      install: source linux/install.sh
+  - os: linux
+    install: source linux/install.sh
+    script:
+    - conda create -p /tmp/snakestagram --copy -y -q python=3 pandas scikit-learn
+      ipykernel
+    - "tar czf /tmp/snakestagram-v${VERSION}-linux-x64.tar.gz /tmp/snakestagram"
```
Comment (betatim) [inline .travis.yml]: Why oh why does this not get expanded?? Halp

File: linux/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+conda create -p /tmp/snakestagram --copy -y -q python=3 pandas scikit-learn ipykernel
+
+tar czf /tmp/snakestagram-v$VERSION-linux-x64.tar.gz /tmp/snakestagram
```
Comment (rgbkrk) [inline linux/build.sh]: While you're at it, I would `cd` to `/tmp` and make this tarball up `./snakestagram` so it doesn't nest `/tmp`. There may be another way too.

**Comment by betatim** [comment]
This should work now, I think :grin: If the tagging bit works we will find out after merging.

**Comment by betatim** [comment]
I'll merge this so we can test the "deploy on tag", instead of waiting ages for a OSX instance.

**Comment by rgbkrk** [comment]
Works for me, looking forward to it.

**Pull request merged**